### ISSUE TITLE
Update on webpack config

### ISSUE THREAD

**Issue by juhat** [KonnorRogers/bridgetown-automation-tailwindcss#8]
Hi,

this is a wonderful work, thank you for publishing it. 

I found that with the latest bridgetown = 0.21, it does not work properly. I don't know well the webpack config setup, but it seems it might changed at bridgetown side and the generator replaces it with some old values.

If you have time I would be interested to see what wen't wrong.

Thanks!

**Comment by ParamagicDev** [comment]
@juhat I have been meaning to sundown this project. This was used as part of the base for the official bridgetown tailwind plugin.

https://www.bridgetownrb.com/docs/bundled-configurations#tailwindcss

**Comment by andrewmcodes** [comment]
The real question here is how do we make the official documentation more discoverable.

**Comment by juhat** [comment]
Hm, that sounds like a plan. 

I spent 4 hours yesterday to set up a new site with Tailwindcss. I run into several webpack specific rabbit hole, nothing worked. 

I did not find this docs, or at least I did not realised this is important to me.

Still now I started to create a new site 
```
bridgetown new site --use-postcss 
bundle exec bridgetown configure tailwindcss
```

This gives a 
```
[2021-07-01 10:13:05] ERROR `/MISSING_WEBPACK_MANIFEST' not found.
```

after all. I ended up here yesterday too. Any suggestions?

**Comment by andrewmcodes** [comment]
> Hm, that sounds like a plan. 
> 
> 
> 
> I spent 4 hours yesterday to set up a new site with Tailwindcss. I run into several webpack specific rabbit hole, nothing worked. 
> 
> 
> 
> I did not find this docs, or at least I did not realised this is important to me.
> 
> 
> 
> Still now I started to create a new site 
> 
> ```
> 
> bridgetown new site --use-postcss 
> 
> bundle exec bridgetown configure tailwindcss
> 
> ```
> 
> 
> 
> This gives a 
> 
> ```
> 
> [2021-07-01 10:13:05] ERROR `/MISSING_WEBPACK_MANIFEST' not found.
> 
> ```
> 
> 
> 
> after all. I ended up here yesterday too. Any suggestions?

It sounds like you aren't running the webpack dev server. Are you using `yarn start` to run the site?

**Comment by juhat** [comment]
Hi @andrewmcodes 

Yes, I use it. However strangely, I tried these steps Today and now they work. I can't recall what did I do previously differently, I tried like 4-6 times and nothing worked. My only guess is, that I might not restarted "yarn start"? Other than that I have no clue why it works now and did not work last week. Another clue might be a node upgrade...

Thanks for the help!

I can feel all the documentation work and CLI polishing along the way. Even I stuck with this, I enjoyed reading the docs and I found the tooling to be extremely helpful. Thanks for working on it!

**Comment by andrewmcodes** [comment]
> Hi @andrewmcodes 
> 
> 
> 
> Yes, I use it. However strangely, I tried these steps Today and now they work. I can't recall what did I do previously differently, I tried like 4-6 times and nothing worked. My only guess is, that I might not restarted "yarn start"? Other than that I have no clue why it works now and did not work last week. Another clue might be a node upgrade...
> 
> 
> 
> Thanks for the help!
> 
> 
> 
> I can feel all the documentation work and CLI polishing along the way. Even I stuck with this, I enjoyed reading the docs and I found the tooling to be extremely helpful. Thanks for working on it!

@juhat yeah sounds like something was off. A good way to debug is to just open the folder that Bridgetown outputs to. That will give you a better understanding of what is being created behind the scenes, as well as help you spot very quickly when something is missing. 

Whenever I hit a bug that is not immediately obvious, I usually remove all of the generated folders (`yarn clean` if you are using the same script names that Bridgetown gave you) and that fixes it like 95% of the time. And I mean 95% across all static site generators and Webpack setups - not Bridgetown. Almost like Spring 😆 

You definitely cannot leave the Webpack server on indefinitely like you can with a Rails server or a Bridgetown server so I would recommend shutting it off when you are done with it to avoid JS weirdness.

**Comment by andrewmcodes** [comment]
@juhat oh almost forgot! As I'm sure you are aware, documentation is really hard so if you spot something that can be improved and aren't feeling up to it yourself (yet!), you should definitely open an issue on the repo or better yet, hop in the [Bridgetown Discord Server](https://discord.gg/4E6hktQGz4) where there may be someone around to help! If nothing else, feel free to send them to me directly and ill get the feedback where it needs to go :) let us know if we can help more!
